bin/mv.js

```diff
@@ -141,31 +138,118 @@ function loadProfile(name) {
     return config.getProfile(name);
 }
 
-function moveCronJob(profile, name, target) {
+function moveCronJob(profile, name, target, options) {
     return coroutine(function*() {
-        var cronJobs = yield profile.listCronJobs();
-
-        for (let job of cronJobs) {
-            if (job.name === name) {
-                let token = yield profile.getWebtask({
-                    name: target.name
-                });
-                yield target.profile.createCronJob({
-                    name: target.name,
-                    container: target.container,
-                    token: token.token,
-                    schedule: job.schedule
-                });
-                yield profile.removeCronJob({
-                    name: name
-                });
-                break;
+        var job;
+
+        try {
+            job = yield profile.getCronJob({
+                name: name
+            });
+        } catch (err) {
+            if (err.statusCode === 404) {
+                return;
             }
+            throw err;
+        }
+
+        let webtask = yield profile.getWebtask({
+            name: target.name,
+            container: target.container
+        });
+
+        if (_.get(options, 'verify') && job.token !== options.verify) {
+            console.log(Chalk.bold('* Warning: failed to verify the cron job token (no match).'));
         }
+
+        yield target.profile.createCronJob({
+            name: target.name,
+            container: target.container,
+            token: webtask.token,
+            schedule: job.schedule
+        });
+
+        yield profile.removeCronJob({
+            name: name
+        });
+    })();
+}
+
+function copyStorage(webtask, target) {
+    return coroutine(function*() {
+        let targetWebtask = yield target.profile.getWebtask({
+            name: target.name,
+            container: target.container
+        });
+        let data = yield exportStorage(webtask);
```

Comment:
Since `copyStorage` happens after the `source` webtask has been removed, I think that the source data will also already have been removed. I really appreciate the creative solution you brought to this problem but I think the problem might be better solved by something outside the scope of this command.

We will need to consider adding a `?data=true` query parameter to `/api/token/inspect`. I wonder if it would also make sense to add support for passing data in at token creation (@tjanczuk, what do you think?).

The reason I propose this is because it doesn't really make sense to me to require the creation of ephemeral import / export webtasks that will immediately be replaced / removed, respectively.